chapters/classifier.rst

```diff
@@ -0,0 +1,47 @@
+==================
+Classifier in Jina
+==================
+Classifier allows you to perform classification and regression based on given input and have the predicted hard/soft labels as output.
+
+.. contents:: Table of Contents
+    :depth: 3
+
+Motivation
+------------------------------------------
+:class:`Classifier` can be used for document meta data enrichment. :class:`Classifier` may help to generate and add tags to Documents automatically
```

Comment:
i think this reference will generate a warning message when you `make build`, `jina.executors.classifiers.BaseClassifier` should work. Or you simply use `Classifier`